lib/assessment.rb

```diff
@@ -4,37 +4,38 @@ class Assessment
   # Write ruby code to return the longest entry in the
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
-  def longest(enum)
-    # Your code goes here
+  def longest(enum) { |a, b| a.length <=> b.length }
+    enum.max
   end
 
+
   # Question 2
   # Write ruby code to return the sum of entries in the
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.length
```

Comment:
When summing across the elements in an array (or other enumerable), the tool to use is reduce. 
`enum.reduce(&:+)`